src/main/java/com/epam/page/object/generator/adapter/JavaFileWriter.java

```diff
@@ -42,16 +50,63 @@ public void writeFiles(String outputDir, List<JavaClass> javaClasses) throws IOE
      * @throws IOException can be throwing if outputDir path doesn't correct.
      */
     private void writeClass(String outputDir, JavaClass javaClass) throws IOException {
-        JavaFile.builder(javaClass.getPackageName(), buildTypeSpec(javaClass))
-            .build()
-            .writeTo(Paths.get(outputDir));
+        JavaFile javaFile = getJavaFileFromClass(javaClass);
+        try {
+            String formattedFile = new Formatter().formatSource(javaFile.toString());
+            Path path = this.getFullPath(Paths.get(outputDir), javaFile);
+            Files.write(path, formattedFile.getBytes());
+        } catch (FormatterException e) {
+            logger.warn("There was an error during format a source: " + e.getMessage());
```

Comment:
Do you really need to suppress stack trace?

Reply:
no, was made accidently, will fix